.travis/docker-compose-travis.yml

```diff
@@ -3,8 +3,22 @@ version: "2"
 
 services:
   elasticsearch:
-    image: elasticsearch:1.5.2
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.8.0
```

Comment:
Why we are going to use the image from `docker.elastic` registry. Why not https://hub.docker.com/_/elasticsearch?tab=tags? Are we not sure in the docker hub registry?

Reply:
You are right, done